src/App.jsx

```diff
@@ -11,6 +11,7 @@ import EmptyWishlistViewProducts from "./pages/EmptyWishlistViewProducts";
 import DeleteProduct from "./components/molecules/deleteProduct";
 import CreateAccount from "./pages/CreateAccount";
 import Help from "./pages/Help";
```

Comment:
Also kindly avoid making changes to the App.js file. Routes should only be added to the Routes.js file.